src/lang/en-US.json

```diff
@@ -1213,7 +1214,18 @@
   "activated": "Activated",
   "rune": "Rune",
   "placement": "Placement",
-  "exclude_turbo_matches": "Exclude Turbo matches"
+  "exclude_turbo_matches": "Exclude Turbo matches",
+
+  "scenarios_item_timings": "Item Timings",
+  "scenarios_misc": "Misc",
+  "scenarios_time": "Time",
+  "scenarios_item": "Item",
+  "scenarios_game_duration": "Game Duration",
+  "scenarios_scenario": "Scenario",
+  "scenarios_first_blood": "Team drew First Blood",
+  "scenarios_courier_kill": "Team sniped the enemy courier before the 3-minute mark",
```

Comment:
string is too long for the container (you can increase the width or shorten the string)